compiler/annotation/type_seeder.rs

```diff
@@ -190,7 +190,7 @@ impl<'this, Snapshot: ReadableSnapshot> TypeGraphSeedingContext<'this, Snapshot>
             disjunction.conjunctions().iter().map(|conj| self.build_recursive(context, conj)).collect_vec();
         let shared_variables = disjunction
             .referenced_variables()
-            .filter(|var| context.is_variable_available(parent_conjunction.scope_id(), *var))
+            .filter(|var| context.is_in_scope_or_parent(parent_conjunction.scope_id(), *var))
```

Comment:
I don't think these kind of renames help. Yes, sure it's checking whether the variable is in the current scope or parent. If I wanted to know what this line is doing, I could always get that information from clicking through. What I'm not getting from the new name is why it's doing it, and I'm left guessing. 
I do actually have trouble following the logic when we rename things like this. It's exposing the implementation details and lowering the level of abstraction.
I think it's true that a variable is available iff it's declaration is considered to be in this or a parent scope, so I prefer the old name. 
I felt the same about Dmitrii's use of scope "transparency". It makes for an easier read because it conveys the relevance of the function, rather than summarising it's implementation.